src/modules/cpu/kernel/brightness.hpp

```diff
@@ -93,7 +103,14 @@ RppStatus brightness_u8_u8_host_tensor(Rpp8u *srcPtr,
                 int vectorLoopCount = 0;
                 for (; vectorLoopCount < alignedLength; vectorLoopCount += vectorIncrement)
                 {
-#if __AVX2__
+#if __AVX512__
+                    __m512 p[6];
+                    rpp_simd_load(rpp_load96_u8pkd3_to_f32pln3_avx512, srcPtrTemp, p);
+                    compute_brightness_96_host(
+                        p, pBrightnessParams);  // brightness adjustment
+                    rpp_simd_store(rpp_store96_f32pln3_to_u8pln3_avx512, dstPtrTempR,
+                                    dstPtrTempG, dstPtrTempB, p);  // simd stores
```

Comment:
Please align the function params
It should be in single line
compute_brightness_96_host(p, pBrightnessParams);  // brightness adjustment

Applicable to all such instances in the file